dev-docs/howtoguides/how_to_setup_windows_machine_for_wsl_testing.md

```diff
@@ -0,0 +1,158 @@
+# How to setup Windows machine for WSL testing
+
+To run our integration tests on WSL instances, we need to first setup a Windows machine
+to allows us to launch the WSL instances. Not only that, we also need to setup OpenSSH
+server to allow us to run ssh commands on the Windows machine.
+
+This how to guide will explain all the necessary steps for the setup, from launching the image
+from Azure to testing if WSL is correctly working on the machine.
```

Comment:
```suggestion
This how-to guide will explain all the necessary steps for the setup, from launching the image
in Azure to testing if WSL is correctly working on the machine.
```

Reply:
I guess this is a matter of taste, but wouldn' tit read better if we drop the `how-to` part of the sentence : `This how-to guide will explain all the necessary step...`. Also not being an explanation, but rather a how-to guide, shouldn't we rephrase that sentence into `This guide presents all the necessary steps...` ?

Reply:
Yes, it is the latter. We are launching a Windows machine in Azure. And I will also specify the Windows version here as well